src/einsteinpy/plotting/geodesics_scatter.py

```diff
@@ -4,21 +4,40 @@
 from matplotlib.animation import FuncAnimation
 
 from einsteinpy.metric import Schwarzschild
+from einsteinpy.utils import schwarzschild_radius
 
 
 class ScatterGeodesicPlotter:
     """
     Class for plotting static matplotlib plots.
+
```

Comment:
Why are parameters specified in class docstring?

Reply:
@ I didn't know it had to be in the rst. will shift them

Reply:
No, I never said checks fail due to this :)